File: Python/Burbuja.py
import random
arr = []
for i in range(10):
    arr.append(random.randint(1,100))

print("El arreglo sin ordenarse es: ", arr)
for i in range(len(arr)):
    for j in range(len(arr) - i -1):
        if arr[j] > arr[j+1]:
            temp = arr[j]
            arr[j] = arr[j+1]
            arr[j+1] = temp

print("El arreglo ya ordenado es: ", arr)

File: JavaScript/Burbuja.js
max = 100
min = 1
arr = []

for (let i = 0; i < 10; i++){
    let n =  Math.floor(Math.random() * (max - min + 1)) + min
    arr[i] = n
}
console.log("El array desordenado es: ")
console.log(arr.join(", "));
console.log("\n")


for (let i = 0; i < arr.length; i++){
    for(let j = 0; j < arr.length-1-i; j++){
        if (arr[j] > arr[j+1]){
            let temp = arr[j]
            arr[j] = arr[j+1]
            arr[j+1] = temp
        }
    }
}

console.log("El arreglo ordenado es: ")
console.log(arr.join(", "));

